packages/js/uri-resolvers/src/__tests__/wrapper-resolver/histories/can-resolve-a-wrapper.json

```diff
@@ -0,0 +1,3 @@
+[
+  "wrap://test/wrapper => Wrapper (wrap://test/wrapper) => wrapper (wrap://test/wrapper)"
```

Comment:
Love these smaller more specific test files! 👏

Reply:
Much easier to maintain and they even caught a minor bug!